Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Rage;
+using Rage.Native;
+
+namespace RageCommunity.Library.Wrappers
+{
+    public static partial class NativeWrappers
+    {
+        /// <summary>
+        /// Causes the given ped to start playing the specified scenario.  A list of scenarios can be found <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/scenariosCompact.json">here</a>.
+        /// </summary>
+        public static void TaskStartScenarioInPlace(Ped ped, string scenarioName, int unkDelay = 0, bool playEnterAnimation = true)
+        {
+            NativeFunction.Natives.TASK_START_SCENARIO_IN_PLACE(ped, scenarioName, 0, playEnterAnimation);
+        }
+
+        /// <summary>
+        /// Causes the given ped to face the specified entity for a duration (in ms).
+        /// </summary>
+        public static void TaskTurnPedToFaceEntity(Ped ped, Entity entity, int duration)
+        {
+            NativeFunction.Natives.TASK_TURN_PED_TO_FACE_ENTITY(ped, entity, duration);
```

Comment:
Do we know how this task can be started and stopped instead of being time-based? Does a negative duration like -1 mean that the ped will keep this task?

Reply:
Yes, -1 works as infinite